handbook/engineering/commit_messages.md

```diff
@@ -0,0 +1,39 @@
+# Commit message guidelines
+
+Having consistent commit messages makes it easier to see at a glance what code has been affected by recent changes. It also makes it easier to search through our commit history.
+
+To get some background, please read [How to Write a Git Commit Message](https://chris.beams.io/posts/git-commit/)
+
+## General guidelines
+
+1. Separate subject from body with a blank line
+2. Limit the subject line to 50 characters
+3. Capitalize the subject line
+4. Do not end the subject line with a period
+5. Use the imperative mood in the subject line
+6. Wrap the body at 72 characters
```

Comment:
I use the GitHub Desktop app which doesn't provide a way to configure this (https://github.com/desktop/desktop/issues/62). Even if I didn't, I am personally not a fan of hard wrapping stuff like this because I would rather have the display do the wrapping.

Reply:
Agreed, I'm happy to remove that point